.github/workflows/broker.yml

```diff
@@ -155,16 +171,19 @@ jobs:
           flavor: |
             latest=auto
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1.1.1
+        id: buildx2
+        uses: docker/setup-buildx-action@v1.5.1
         with:
           driver-opts: |
             network=host
+      - name: available platforms
+        run: echo ${{ steps.buildx2.outputs.platforms }}
       - name: Build
-        uses: docker/build-push-action@v2.3.0
+        uses: docker/build-push-action@v2.7.0
         with:
           context: .
           file: ./Dockerfile
-          platforms: linux/amd64
+          platforms: linux/amd64,linux/arm64
```

Comment:
If tests above were successful then Docker should only build `linux/arm64` image at this point.

Reply:
but will it then not also only push the arm version?